runtime/transformations/components/data-quality/value_range_check.py

```diff
@@ -139,13 +139,11 @@ class ValueRange(BaseModel):
 
     @root_validator()
     def verify_value_ranges(cls, values: dict) -> dict:
-        try:
-            min_value = values["min_value"]
-            min_value_inclusive = values["min_value_inclusive"]
-            max_value = values["max_value"]
-            max_value_inclusive = values["max_value_inclusive"]
-        except KeyError as error:
-            raise ValueError("") from error
+        min_value = values["min_value"]
+        min_value_inclusive = values["min_value_inclusive"]
+        max_value = values["max_value"]
+        max_value_inclusive = values["max_value_inclusive"]
+
```

Comment:
Wouldn't it be better to add an expressive error message?

Reply:
I'll use skip_on_failure instead.